notification_configuration.go

```diff
@@ -93,12 +92,27 @@ type NotificationConfiguration struct {
 
 // DeliveryResponse represents a notification configuration delivery response.
 type DeliveryResponse struct {
-	Body       string      `json:"body"`
-	Code       int         `json:"code"`
-	Headers    http.Header `json:"headers"`
-	SentAt     time.Time   `json:"sent-at,iso8601"`
-	Successful bool        `json:"successful"`
```

Comment:
This never worked as intended. It would default to `false`. See https://play.golang.org/p/u6bGq49tHa7